cluster/examples/kubernetes/ceph/crds.yaml

```diff
@@ -1359,6 +1359,8 @@ spec:
                   type: object
                   nullable: true
                   x-kubernetes-preserve-unknown-fields: true
+                objectBucketName:
```

Comment:
You also need to update `cluster/charts/rook-ceph/templates/resources.yaml` and `tests/framework/installer/ceph_manifests.go`.